apps/keyboard/js/imes/latin/latin.js

```diff
@@ -235,6 +250,28 @@
     // Start off with the correct capitalization
     updateCapitalization();
 
+    // Get user dictionary blob.
+    getUserDictionaryBlobPromise =
+      dbStore.getItem('dictblob').then(function(blob) {
+      // If worker is there, tell it we have a (new) user dictionary blob,
+      // since setLanguage() may be bypassed due to language not changing.
+      // (User dictionary is langauge-neutral.)
+      // Do pass the argument if blob is undefined (no user dictionary words),
+      // Since the worker needs to know if user has deleted all words.
+      // Worker will bypass UserDictionary prediction as needed.
+      // |blob| may be |undefined| so we don't Transfer it yet.
```

Comment:
How about we do `typeof blob` check here to decide if we want to use Transfer? I don't know if it's worth it though.